### DISCUSSION TITLE
Creating sub-lists within the dialogs

### DISCUSSION THREAD

**Discussion by GusGF** [freeplane/freeplane#552] [Help] [answered]
Folks,
How do I create a sub-list within a list when I'm editing a node core in the dialog box using a keyboard shortcut? The screen grab below was achieved by editing the HTML.

![image](https://user-images.githubusercontent.com/21323227/183887801-88e0e102-65ba-4051-acaf-5bad635f66ae.png)

**Comment by euu2021** [comment, marked as answer]
I see that on `Edit->Node core->Edit node core in dialog`, the Tab does the job, but on inline edit it doesn't. Possibly related: #511

**Reply by euu2021** [reply]
I see that you may also be asking about how to assign a hotkey for this:

![image](https://user-images.githubusercontent.com/77707706/183911907-6b831a60-85af-49fb-9420-fdc2934383a9.png)



In that case, it's not possible. [See this](https://sourceforge.net/p/freeplane/bugs/3008/#08c9).

**Reply by GusGF** [reply]
Thank you again tab & shift tab do the job.
